libs/blocks/share/share.js

```diff
@@ -37,35 +37,58 @@ export async function getSVGsfromFile(path, selectors) {
 
 function getPlatforms(el) {
   const manualShares = el.querySelectorAll('a');
-  if (manualShares.length === 0) return null;
-  return [...manualShares].map((link) => {
-    const { href } = link;
+  if (manualShares.length === 0) return ['facebook', 'twitter', 'linkedin', 'pinterest', 'reddit'];
+  const platforms = [];
+  [...manualShares].forEach((share) => {
+    const { href } = share;
     const url = new URL(href);
     const parts = url.host.split('.');
-    return parts[parts.length - 2];
+    platforms.push(parts[parts.length - 2]);
+    const parentP = share.closest('p');
+    parentP?.remove();
   });
+  return platforms;
 }
 
 export default async function decorate(block) {
   const config = getConfig();
   const base = config.miloLibs || config.codeRoot;
-  const platforms = getPlatforms(block) || [
-    'facebook',
-    'twitter',
-    'linkedin',
-    'pinterest',
-    'reddit',
-  ];
-  block.innerHTML = '';
+  const platforms = getPlatforms(block);
+  const rows = block.querySelectorAll(':scope > div');
+  const childDiv = rows[0].querySelector(':scope > div');
+  const emptyRow = childDiv?.innerText.trim() === '';
+  /* eslint-disable  no-confusing-arrow,no-useless-escape */
+  const toSentenceCase = (str) => str && typeof str === 'string' ? str.toLowerCase().replace(/(^\s*\w|[\.\!\?]\s*\w)/g, (c) => c.toUpperCase()) : '';
```

Comment:
nit: If we're already disabling eslint for this line, do you think we could break it out into multiple lines for better readability?

Reply:
Ah yep. Good call. 
I added the `/* eslint-disable-next-line no-useless-escape */` part 
(I tested removing those 'useless' escapes and it didn't 'Sentence Case' right so I left that part as is) 
I adjusted the arg into a few lines for better readability and fixes 'no-confusing-arrow' clause. :) 
Thanks for the feedback